src/Image/index.d.ts

```diff
@@ -3,6 +3,7 @@ import * as React from 'react';
 export type ImageFit = 'contain' | 'cover' | 'tile' | 'none';
 
 export interface ImageProps extends React.ImgHTMLAttributes<HTMLImageElement> {
+  borderRadius? : string,
```

Comment:
If the type is pure CSS property, we use the direct type for it:

```suggestion
  borderRadius?: React.CSSProperties['borderRadius'];
```

Reply:
done